javascript-data-model/order-history.js

```diff
@@ -0,0 +1,102 @@
+var item = { };
+var person = 'JS Masher';
+var orderHistory = [
+  item.orderOne = {
```

Comment:
This is not how a piece of data is added to an array. The assignment operator `=` will not be present here, the data in an array is just a list.